project/assets/login.less

```diff
@@ -0,0 +1,90 @@
+.body-structure {
+    width: 100%;
+}
+
+.body-skin {
+    background-color: rgb(230, 230, 230);
+}
+
+header {
+    background: url("images/feedback-banner.jpg") center center no-repeat;
+    opacity: 0.8;
+    height: 600px;
+    width: 100%;
+
+    .grid {
+        height: 100%;
+    }
+}
+
+.login-section-1 {
+    padding: 70px;
+}
+
+.login-section-1-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-section-text {
+    padding: 70px;
+}
+
+.login-section-2 {
+    padding: 30px;
+}
+
+.login-form {
+    align-self: center;
+    padding-bottom: 30px;
+    margin-left: 20%;
+    width: 25%;
+    height: 65%;
+}
+
+.login-form-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-form-items {
+    padding: 25px 25px 5px;
+}
+
+.login-form-header-skin {
+    background-color: rgb(51, 153, 255);
+    color: rgb(255, 255, 255);
+    font-family: roboto;
+    font-weight: bold;
+}
+
+.login-form-header-text {
+    margin-top: 0%;
+    line-height: 45px;
+    text-align: center;
+}
+
+input[type=text],
+input[type=password] {
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid rgb(166, 166, 166);
+    font-family: roboto;
+    font-size: 20px;
+    padding-bottom: 10px;
+    padding-top: 10px;
+}
+
+.login-form-button {
+    box-shadow: 10px 35px -9px rgba(204, 194, 204, 1);
+    line-height: 30px;
+    width: 25%;
+    margin-top: 42%;
+    margin-left: 33%;
+}
+
+.login-form-button-skin {
+    background-color: rgb(51, 153, 255);
```

Comment:
general feedback: You should use 'less' variables for ALL colors. Maybe define them in a colors.less file and import them here.